google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -250,6 +258,23 @@ class ConnectionWorker implements AutoCloseable {
   private static String projectMatching = "projects/[^/]+/";
   private static Pattern streamPatternProject = Pattern.compile(projectMatching);
 
+  private Meter writeMeter;
+  static AttributeKey<String> telemetryKeyStreamId = AttributeKey.stringKey("streamId");
```

Comment:
Is streamId here meaningful?

Reply:
I have removed this metric for now. Stream-level information may be too specific and result in a high cardinality for this attribute. What do you think of including an attribute that is the table name? This would be "projects/<project name>/dataset/<dataset name>/table/<table name>". This way a customer could inspect traffic metrics for a particular table, and compare it against other tables.